.eslintrc.js

```diff
@@ -209,6 +210,46 @@ module.exports = {
               "Do not use @vue/test-utils' `shallowMount`. Use `~~/test/unit/test-utils/render` instead which includes helpful context setup or @testing-library/vue's `render` directly.",
           },
         ],
+        "@intlify/vue-i18n/no-raw-text": "off",
```

Comment:
```suggestion
```

Duplicated, see the second line of the "rules" block

Reply:
Oops, thanks!